src/lib/tabs/tab-nav-bar/tab-nav-bar.scss

```diff
@@ -23,6 +23,14 @@
     flex-basis: 0;
     flex-grow: 1;
   }
+
+  &.mat-tab-disabled {
+    // We use `pointer-events` to make the element unclickable when it's disabled, rather than
+    // preventing the default action through JS, because we can't prevent the action reliably
+    // due to other directives potentially registering their events earlier. This shouldn't cause
+    // the user to click through, because we always have a `.mat-tab-links` behind the link.
+    pointer-events: none;
```

Comment:
Will this prevent tabbing to the element and hitting `enter` to navigate?

Reply:
We add `tabindex="-1"` when the link becomes disabled so you can't tab to it.